### DISCUSSION TITLE
2 questions regarding configration

### DISCUSSION THREAD

**Discussion by ghost** [nvim-tree/nvim-tree.lua#2260] [Q&A]
Hi! this plugin is nice and looks great when trying to turn nvim into an ide 

however im confused about 2 things that people have in their screenshots that i cant seem to replicate:

1. how do i remove the line on the left of my editing buffer?
( this one, sorry for cam quality lmao: 
![](https://i.imgur.com/hKKjklZ.jpg)
)

2. how do i make it so when i hit enter on a selected file, it automatically opens it in a new tab?

thanks for your help!

**Comment by alex-courtis** [comment]
> how do i remove the line on the left of my editing buffer?

That's out of nvim-tree's control, neovim itself controls window borders. See `:help fillchars` and `:help WinSeparator`.

> how do i make it so when i hit enter on a selected file, it automatically opens it in a new tab?

Map enter to `api.node.open.tab`, see `:help nvim-tree-mappings`.

I _strongly_ advise against doing that; vim is not an ide and vim tabs have a very different purpose. See https://github.com/nvim-tree/nvim-tree.lua/discussions/1598 for a full discussion.
